tests/serialization/test_arrow_compat.py

```diff
@@ -53,4 +52,7 @@ def test_arrow_compat(arrow_version, reference_store, mocker):
     if arrow_version in ("0.14.1", "0.15.0", "0.16.0") and not ARROW_LARGER_EQ_0141:
         orig = orig.astype({"null": float})
 
+    if LooseVersion(arrow_version) < "0.13.0":
```

Comment:
can you add a short comment on the reason? Is this because timezones cannot be preserved with these old versions?

Reply:
Because the changes from @ged-steponavicius fixes https://github.com/JDASoftwareGroup/kartothek/issues/259, enabling proper compatibilty of datetimes with timezones i.e. these columns, and we currently only support pyarrow >= 0.13.0

Reply:
Then why do we need to check `< "0.13.0":` That's dead code, isn't it?

Reply:
`arrow_version` here refers to the arrow version with which the reference data file was created

Reply:
```suggestion
    if LooseVersion(arrow_version) < "0.13.0":  # gh-259: compatibilty of datetimes with timezones only supported in kartothek versions with pyarrow >= 0.13.0
```

Reply:
suggestion looks good to me, thanks :)